test/unit/models/users.test.js

```diff
@@ -87,5 +87,62 @@ describe("users", function () {
       expect(user.last_name).to.equal(userData.last_name);
       expect(userExists).to.equal(true);
     });
+
+    it("should add the github_user_id to the user collection", async function () {
+      const userData = userDataArray[0];
+      userData.github_user_id = "12345678";
+
+      const { isNewUser, userId } = await users.addOrUpdate(userData);
+
+      const data = (await userModel.doc(userId).get()).data();
+
+      expect(data.github_user_id).to.equal(userData.github_user_id);
+      expect(isNewUser).to.equal(true);
+    });
+
+    it("should update the github_id in the user collection", async function () {
+      const userData = userDataArray[0];
+      userData.github_user_id = "Yash Sinha";
```

Comment:
I think this will be `userData.github_id`. Please check

Reply:
Yes it should be `github_id` thanks for pointing this out.